WordPress/AbstractClassRestrictionsSniff.php

```diff
@@ -51,8 +53,9 @@
 	 * Documented here for clarity. Not (re)defined as it is already defined in the parent class.
 	 *
 	 * @return array
+	 *
+	abstract public function getGroups();
```

Comment:
What do you mean ?
> _"Not (re)defined as it is already defined in the parent class."_

Comments style wise if I close the doc block and use a `//` comment, I get a notice that inline comments need a blank line before them, so this way, the abstract method is still documented in the child abstract class *and* passes the doc checks.

There may be better ways to do this, but I couldn't think of any. Advice welcome ;-)